### PR TITLE
Re-organize scripts and CI config files

### PR DESCRIPTION
**Pull request by romac**
## Description

- Move scripts at root of repo into `scripts` folder
- Rename `setup_relayer.sh` script to `e2e.sh`
- Move `docker-compose.yml` under `ci` directory
- Fix whitespace in `ci/README.md`

______

For contributor use:

- [ ] Updated the __Unreleased__ section of [CHANGELOG.md](https://github.com/informalsystems/ibc-rs/blob/master/CHANGELOG.md) with the issue.
- [ ] If applicable: Unit tests written, added test to CI.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation (`docs/`) and code comments.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: ci/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This folder contains the files required to run the End to end testing in [Github
 
 ## End to end (e2e) testing
 
-The [End to end (e2e) testing workflow](https://github.com/informalsystems/ibc-rs/actions?query=workflow%3A%22End+to+End+testing%22) spins up two `gaia` chains (`ibc-0` and `ibc-1`) in Docker containers and one container that runs the relayer. There's a script that configures the relayer (e.g. configure light clients and add keys) and run transactions and queries. A successful run of this script ensures that the relayer is working properly with two chains that support `IBC`. 
+The [End to end (e2e) testing workflow](https://github.com/informalsystems/ibc-rs/actions?query=workflow%3A%22End+to+End+testing%22) spins up two `gaia` chains (`ibc-0` and `ibc-1`) in Docker containers and one container that runs the relayer. There's a script that configures the relayer (e.g. configure light clients and add keys) and run transactions and queries. A successful run of this script ensures that the relayer is working properly with two chains that support `IBC`.
```
Comment (ancazamfir) [inline ci/README.md:7]:
```suggestion
The [End to end (e2e) testing workflow](https://github.com/informalsystems/ibc-rs/actions?query=workflow%3A%22End+to+End+testing%22) spins up two `gaia` chains (`ibc-0` and `ibc-1`) in Docker containers and one container that runs the relayer. There's a script that configures the relayer (e.g. configure light clients and add keys) and runs transactions and queries. A successful run of this script ensures that the relayer is working properly with two chains that support `IBC`.
```

File: ci/README.md [hunk 2]
```diff
@@ -16,52 +16,52 @@ __Note__: This assumes you are running this the first time, if not, please ensur
...
     Creating relayer ... done
     ```
-   
+
     If you want to ensure they are all running properly you can use the command:
-   
+
     `docker-compose ps`
-   
+
```
Comment (ancazamfir) [inline ci/README.md:44]: `docker-compose -f ci/docker-compose.yml ps` above and
`docker-compose -f ci/docker-compose.yml down` further down.

**Review by ancazamfir — COMMENTED** [review]
Looks good, thanks Romain! A couple of minor comments.

File: ci/README.md [hunk 3]
```diff
@@ -16,52 +16,52 @@ __Note__: This assumes you are running this the first time, if not, please ensur
...
     Creating ibc-1 ... done
     Creating ibc-0 ... done
     Creating relayer ... done
     ```
-   
+
     If you want to ensure they are all running properly you can use the command:
-   
+
     `docker-compose ps`
```
Comment (romac) [inline ci/README.md:43]:
```suggestion
    `docker-compose -f ci/docker-compose.yml ps`
```

**Review by ancazamfir — APPROVED** [review]

**Pull request merged**